backend/typescript/swagger.yml

```diff
@@ -32,232 +32,6 @@ components:
-                properties:
-                  intField:
-                    type: integer
-                  stringField:
-                    type: string
-                  stringArrayField:
-                    type: array
-                    items:
-                      type: string
-                  enumField:
-                    type: string
-                    enum: ["A", "B", "C", "D"]
-                  boolField:
-                    type: boolean
-                  fileName:
-                    type: string
-        '401':
-          description: Unauthorized
-        '500':
-          description: Invalid request
-  
-  
-  /entities/{id}:
-    get:
-      security:
-        - bearerAuth: []
-      parameters:
-        - in: path
-          name: id
-          required: true
-          schema:
-            # type: integer (postgres)
-            # minimum: 1
-            type: string
-          description: The entity ID
-      tags:
-        - Entity
-      description: Returns an entity based on its ID
-      responses:
-        '200':
-          description: Successfully returns entity based on ID
-          content:
-            application/json:
-              schema:
-                type: object
-                required:
-                  - intField, stringField, boolField, enumField, stringArrayField
-                properties:
-                  intField:
-                    type: integer
-                  stringField:
-                    type: string
-                  stringArrayField:
-                    type: array
-                    items:
-                      type: string
-                  enumField:
-                    type: string
-                    enum: ["A", "B", "C", "D"]
-                  boolField:
-                    type: boolean
-                  fileName:
-                    type: string
-        '401':
-          description: Unauthorized
-        '500':
-          description: Invalid ID
-
-
-    put:
-      security:
-        - bearerAuth: []
-      parameters:
-        - in: path
-          name: id
-          required: true
-          schema:
-            # type: integer (postgres)
-            # minimum: 1
-            type: string
-          description: The entity ID
-      tags:
-        - Entity
-      description: Edit an entity based on its ID
-      requestBody:
-        required: true
-        content:
-          multipart/form-data:
-            schema:
-              type: object
-              properties:
-                body:
-                  type: string
-                file:
-                  type: string
-                  format: binary
-      responses:
-        '200':
-          description: Successfully edited entity based on ID
-        '401':
-          description: Unauthorized
-        '500':
-          description: Invalid request
-
-
-    delete:
-      security:
-        - bearerAuth: []
-      parameters:

```

Comment:
Can you actually keep the entity documentation for now? We'll remove these as a part of the review service ticket because we modify entity routes there. So just undelete the entity parts